src/ui/src/components/workflows/abstract/WorkflowsNode.vue

```diff
@@ -26,7 +49,112 @@ export default {
 };
 </script>
 <script setup lang="ts">
-import { ComponentPublicInstance, inject, ref } from "vue";
+import { computed, inject } from "vue";
+import { Component } from "@/writerTypes";
+import injectionKeys from "@/injectionKeys";
+
+const wf = inject(injectionKeys.core);
+const componentId = inject(injectionKeys.componentId);
+
+const def = computed(() => {
+	const component = wf.getComponentById(componentId);
+	return wf?.getComponentDefinition(component.type);
+});
+
+const emit = defineEmits(["outSelect"]);
+
+function handleOutClick(ev: DragEvent, outId: string) {
+	ev.stopPropagation();
+	emit("outSelect", outId);
+}
 </script>
 
-<style scoped></style>
+<style scoped>
+.WorkflowsNode {
+	background: var(--builderBackgroundColor);
+	border-radius: 12px;
+	width: 240px;
+	position: absolute;
+	box-shadow: 0px 2px 0px 0px rgba(0, 0, 0, 0.03);
```

Comment:
Don't you want to reuse `var(--containerShadow)`
https://github.com/writer/writer-framework/blob/8c3290cbb4c3e0c4bb233dd8118e1c71bd89efab/src/ui/src/renderer/ComponentRenderer.vue#L96
```suggestion
	box-shadow: var(--containerShadow);
```